CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## HEAD
 
 * Fancier README
+* Add ability for `:on` argument to accept a `Hash` where the keys are exception types and the values are a single or array of `Proc` that evaluates whether to retry exception.
```

Comment:
would be a 3.2.0 release probable

Reply:
I thought `HEAD` would be changed to master on the changelog only when the owner would release it. Should I put it there now?

Reply:
Yeah, I like the changelog stuff in PRs to go into HEAD. It allows me to decide how many commits to group into a release and also lets me decide whether it's a major/minor/patch level version change.

Reply:
yeah i've actually adopted the `HEAD` approach myself at this point, so this was more a way to propose a version number when stuff is released eventually than any actual complaint

Reply:
trivial but "are a single or array" is not quite english.  "and the values are either `Proc`s or arrays of `Proc`s that will evaluate to `true` when the exception should be retried"